app/controllers/application_controller.rb

```diff
@@ -1,3 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def current_tenant
+    current_user&.tenant
+  end
+
+  def require_admin!
```

Comment:
`authorize_admin!` would sound better, and it's not a good place for such methods